packages/core/test/todo.md

```diff
@@ -9,9 +9,8 @@ When implementing the tests for the delete, backspace and enter keys, certain te
         1. [Unbreakable](#f-t-unbreakable)
         2. [PRE](#f-t-pre)
         3. [Tables](#f-t-tables)
-        4. [Pictograms](#f-t-pictograms)
-        5. [HR](#f-t-hr)
-        6. [Integration](#f-t-integration)
+        4. [HR](#f-t-hr)
+        5. [Integration](#f-t-integration)
```

Comment:
Markdown is bad if you truly have to manually re-order every item of the list when you remove one in the middle.